src/BotPlutusInterface/Balance.hs

```diff
@@ -80,17 +85,17 @@ balanceTxIO pabConf ownPkh unbalancedTx =
       privKeys <- newEitherT $ Files.readPrivateKeys @w pabConf
       let utxoIndex = fmap Tx.toTxOut utxos <> unBalancedTxUtxoIndex unbalancedTx
           requiredSigs = map Ledger.unPaymentPubKeyHash $ Map.keys (unBalancedTxRequiredSignatories unbalancedTx)
+
       tx <-
-        hoistEither $
-          addValidRange
-            pabConf
+        newEitherT $
+          addValidRange @w
             (unBalancedTxValidityTimeRange unbalancedTx)
             (unBalancedTxTx unbalancedTx)
 
       lift $ printBpiLog @w Debug $ viaShow utxoIndex
 
       -- We need this folder on the CLI machine, which may not be the local machine
-      lift $ createDirectoryIfMissingCLI @w False (Text.unpack pabConf.pcTxFileDir)
+      lift $ createDirectoryIfMissingCLI @w False (Text.unpack "pcTxFileDir")
```

Comment:
Why are we losing the config here?

Reply:
Not sure how this happened. Probably some "handy" refactoring by IDE I used. Reverted back.